### PR TITLE
fix: button hover color

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/utils/components/Layout.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const globalDocStyle = (vendorSettings: VendorSettings) => {
   return `
     body {
       --primary-color: ${style.primaryColor};
-      --secondary-hover: ${style.primaryHoverColor};
+      --primary-hover: ${style.primaryHoverColor};
```
Comment (danielgent) [inline src/utils/components/Layout.tsx:20]: This matches the actual CSS variables we have and use :facepalm: 

I'm guessing it was some chatGPT introduced issue

Comment (danielgent) [reply src/utils/components/Layout.tsx:20]: Do we want to snapshot a button being hovered? I think I've done this in playwright before...  I'm not sure if we can do get the browser to hover over a button and then snapshot though...

Now this is working it should _just work_

Comment (danielgent) [reply src/utils/components/Layout.tsx:20]: Tested locally and works :+1:

**Review by markusahlstrand — APPROVED** [review]
LGTM!

**Pull request merged**